amendment/sections/03-terms_defs.adoc

```diff
@@ -0,0 +1,65 @@
+
+[[terms]]
+== Terms, definitions and abbreviations
+
+=== Terms and definitions
+
+[[calendar-spam]]
```

Comment:
Remove all anchors from Terms section and use `term:[]` command.

Reply:
Fixed